indico/modules/designer/models/templates.py

```diff
@@ -64,9 +65,8 @@ class DesignerTemplate(db.Model):
         JSON,
         nullable=False
     )
-    background_image_id = db.Column(
```

Comment:
Well this started when I created the new JSON structure where you could configure the background for both sides. Instead of adding a second column for the backside background, I updated the JSON structure to support the same structure for both back and front side. Afterwards we changed out mind and added the the backside_template column.
We can still load the templates that use a specific background image (`DesignerImageFile.template`), so I don't see a harm doing it this way.
@pferreir can you confirm that this is what we discussed?